app/src/Sentry.web.ts

```
@@ -0,0 +1,60 @@
+    dsn: SENTRY_DSN,
+    debug: false,
+    // Performance Monitoring
+    tracesSampleRate: 1.0,
+    // Session Replay
+    replaysSessionSampleRate: 0.1,
+    replaysOnErrorSampleRate: 1.0,
+    // Disable collection of PII data
+    beforeSend(event) {
+      // Remove PII data
+      if (event.user) {
+        delete event.user.ip_address;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)